sources/packages/web/src/views/institution/OfferingsUpload.vue

```diff
@@ -0,0 +1,316 @@
+            :rows="DEFAULT_PAGE_LIMIT"
+            :rowsPerPageOptions="PAGINATION_LIST"
+            :loading="loading"
+            breakpoint="1250px"
+          >
+            <Column header="Line" field="recordLineNumber"></Column>
+            <Column header="Location" field="locationCode"></Column>
+            <Column header="Program code" field="sabcProgramCode"></Column>
+            <Column
+              header="Start date"
+              field="startDateFormatted"
+              bodyClass="text-no-wrap"
+            ></Column>
+            <Column
+              header="End date"
+              field="endDateFormatted"
+              bodyClass="text-no-wrap"
+            ></Column>
+            <Column header="Status"
+              ><template #body="slotProps">
+                <status-chip-offering
+                  v-if="slotProps.data.offeringStatus"
+                  :status="slotProps.data.offeringStatus"
+                /> </template
+            ></Column>
+            <Column header="Validations"
+              ><template #body="slotProps">
+                <div
+                  class="alert alert-danger"
+                  v-if="slotProps.data.errors.length"
+                >
+                  <ul class="m-2">
+                    <li v-for="error in slotProps.data.errors" :key="error">
+                      {{ error }}
+                    </li>
+                  </ul>
+                </div>
+                <div
+                  class="alert alert-warning"
+                  v-if="slotProps.data.warnings.length"
+                >
+                  <ul class="m-2">
+                    <li
+                      v-for="warning in slotProps.data.warnings"
+                      :key="warning"
+                    >
+                      {{ warning.warningMessage }}
+                    </li>
+                  </ul>
+                </div>
+              </template></Column
+            >
+          </DataTable>
+        </content-group>
+      </content-group>
+    </content-group>
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { ref, computed } from "vue";
+import {
+  OfferingsUploadBulkInsert,
+  DEFAULT_PAGE_LIMIT,
+  PAGINATION_LIST,
+  OfferingStatus,
+  BannerTypes,
+  VForm,
+  FileInputFile,
+} from "@/types";
+import { EducationProgramOfferingService } from "@/services/EducationProgramOfferingService";
+import StatusChipOffering from "@/components/generic/StatusChipOffering.vue";
+import { useSnackBar } from "@/composables";
+import { FileUploadProgressEventArgs } from "@/services/http/common/FileUploadProgressEvent";
+
+const ACCEPTED_FILE_TYPE = "text/csv";
+const MAX_OFFERING_UPLOAD_SIZE = 4194304;
+
+export default {
+  components: {
+    StatusChipOffering,
+  },
+  setup() {
+    const snackBar = useSnackBar();
+    const loading = ref(false);
+    // Only one will be used but the component allows multiple.
+    const offeringFiles = ref([] as FileInputFile[]);
```

Comment:
not a blocker, just sharing, I recently found this ou code base, 
![image](https://user-images.githubusercontent.com/77353155/190233859-1300d11f-2e4c-4231-8311-cdb637943c4a.png)

Reply:
I changed it. It looks better in the mentioned way.